docs/i18n-docs.md

```diff
@@ -0,0 +1,189 @@
+---
+title: Automation tools for i18n and translate
+authors:
+- "@chouchongYHMing" # Authors' github accounts here.
+reviewers:
+- TBD
+approvers:
+- "@warjiang"
+
+creation-date: 2024-08-19
+
+---
+
+# Automation tools for i18n and translate
+
+## Summary
+
+This is an out-of-the-box automated i18n tool designed for the Karmada dashboard, developed primarily from a Chinese environment perspective. It implements the following features:
```

Comment:
I think i18n tools is designed for general purpose, currently it can have greate integration with karmada dashboard. Maybe it can work with karmadactl cli.